src/apis/index.js

```diff
@@ -36,6 +38,12 @@ export function getChangePassword(payload) {
   });
 }
 
+export function getAdmin() {
+  const url = `${API_URL}/${AUTH_API_PREFIX}/showAdminService.json}`;
+  // eslint-disable-next-line camelcase
+  return ajax.get(url, { access_token: cookies.get('loggedIn') });
```

Comment:
The ajax file has been moded so that it will automatically send this access_token value if available, so you don't need to put the acess_token here.

Reply:
Thanks for telling!